Demo/Common/IntegrationExampleDropIn.swift

```diff
@@ -53,14 +58,25 @@ extension IntegrationExample {
     private func handle(_ action: Action) {
         (currentComponent as? DropInComponent)?.handle(action)
     }
+
+    internal func handle(_ order: PartialPaymentOrder) {
```

Comment:
Probably we have to copy those two methods to `IntegrationExampleComponents.swift ` as well